stories/components/select.stories.tsx

```diff
@@ -114,6 +146,15 @@ export function OnlyOptions() {
         filterable={false}
         itemsEqual="label"
         popoverTargetProps={{ style: { display: 'inline-block' } }}
+        popoverProps={{
```

Comment:
Instead of repeating this over and over, please make a hook `useSelect` which contains the state and returns the `items`, `onItemSelect`, default `itemRenderer` and `popoverProps` to be spread into the `Select` component.

If there are any stories that use a filterable select, check if the hook should be adapted to returns something different based on this parameter (and then also return the `filterable` prop in the hook). Check for any other prop that might be useful to return in the hook based on existing stories.

The hook can be exported by react-science to be used by the consumer of the library.

Reply:
implemented